cliquet/events.py

```diff
@@ -47,25 +47,47 @@ def __init__(self, action, timestamp, read_records, request):
 
 
 class ResourceChanged(_ResourceEvent):
-    """Triggered when a resource is changed.
+    """Triggered when a resource is being changed.
     """
     def __init__(self, action, timestamp, impacted_records, request):
         super(ResourceChanged, self).__init__(action, timestamp, request)
         self.impacted_records = impacted_records
 
 
+class AfterResourceRead(_ResourceEvent):
+    """Triggered after a resource was successfully read.
+    """
+    def __init__(self, action, timestamp, read_records, request):
+        super(AfterResourceRead, self).__init__(action, timestamp, request)
+        self.read_records = read_records
+
+
+class AfterResourceChanged(_ResourceEvent):
+    """Triggered after a resource was successfully changed.
```

Comment:
Add a note here explaining that we can be sure it this point that the resource was really changed **and commited** to the database.